--- rosa-media-player/rosa-media-player.desktop.pref~	2013-04-06 23:56:36.330150401 +0200
+++ rosa-media-player/rosa-media-player.desktop	2013-04-06 23:58:32.930150924 +0200
@@ -24,3 +24,4 @@ MimeType=audio/ac3;audio/mp4;audio/mpeg;
 Name=ROSA Media Player
 Type=Application
 X-KDE-StartupNotify=false
+InitialPreference=15
--- rosa-media-player/rosamp_enqueue.desktop.pref~	2013-04-06 23:56:44.173483770 +0200
+++ rosa-media-player/rosamp_enqueue.desktop	2013-04-06 23:58:23.670150883 +0200
@@ -28,3 +28,4 @@ Comment[de]=Datei(en) zur ROSA Media Pla
 Type=Application
 X-KDE-StartupNotify=false
 NoDisplay=true
+InitialPreference=14
